json.c

```diff
@@ -628,18 +628,28 @@ struct json_object *drm_info(char *paths[])
 
 	/* Print everything by default */
 	if (!paths[0]) {
-		char path[PATH_MAX];
-		for (int i = 0;; ++i) {
-			snprintf(path, sizeof path, DRM_DEV_NAME, DRM_DIR_NAME, i);
-			if (access(path, R_OK) < 0)
-				break;
+		drmDevice *devices[64];
```

Comment:
Ah, and see https://gitlab.freedesktop.org/mesa/drm/blame/master/xf86drm.c#L3799 for this one